### PR TITLE
Add AP_Periph Wiki Page

### PR DESCRIPTION
**Pull request by Hwurzburg**
at some point we might want to create a wiki site like plane and copter for Peripheral....even if its small since it would use mostly existing docs

### CODE REVIEW

File: dev/source/docs/ap-peripheral-landing-page.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+- :ref:`Mateksys M8Q<common-matek-m8q>`
+- Hitec GNSS (`Septentrio Mosaic <https://hitecnology.com/drone-peripherals/hcs-positionpro-gnss-receiver>`__ )
+- :ref:`mRo UAVCAN Adapter node<common-mro-uavcan-adapter-node>`
+- :ref:`Orange Cube<common-thecubeorange-overview>` ( it's a flight controller,but it can also be used in AP_Periph..many others to follow)
+
+Firmware
+========
+
+Firmware for existing device definitions are posted `here <https://firmware.ardupilot.org/AP_Periph/>`__. Their hardware definition files are on the ArudPilot GITHUB server, `here <https://github.com/ArduPilot/ardupilot/tree/master/libraries/AP_HAL_ChibiOS/hwdef>`__
```
Comment (khancyr) [inline dev/source/docs/ap-peripheral-landing-page.rst:44]: ArduPilot Github repository

**Review by khancyr — COMMENTED** [review]
Some typos but at least we got something on the wiki ! Nice job

File: dev/source/docs/ap-peripheral-landing-page.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+.. image:: ../images/ap-periph-logo.png
+
+=========
+AP_Periph
+=========
+
+.. image:: ../images/ap-periph-collage.png
+
+
+AP_Periph is an abbreviation for ArduPilot Peripheral, ie. an ArduPilot peripheral device based on the exisitng ArduPilot autopilot code. It takes peripheral device driver libraries of ArduPilot and adapts them to run on stand-alone peripheral devices, which communicate to the main autopilot via CAN, MSP, or other peripheral bus protocols.
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply dev/source/docs/ap-peripheral-landing-page.rst]: thanks

File: dev/source/docs/ap-peripheral-landing-page.rst [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+The Peripheral device usually uses an STMF103 or STMF303 processor, but it is also being ported to use full autopilots as a peripheral also. Both sensors (distance sensor, GNSS, IMU, Barometer, etc.) and output ports (I2C, SPI, PWM, UART, ESC, LED, etc.) can be used to build new peripherals, as well as providing bus expansion for CAN, MSP, I2C,SPI, etc.
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply dev/source/docs/ap-peripheral-landing-page.rst]: done

File: dev/source/docs/ap-peripheral-landing-page.rst (hunk 1 above)
Comment (Hwurzburg) [reply dev/source/docs/ap-peripheral-landing-page.rst]: changed

**Review by tridge — APPROVED** [review]

**Review by magicrub — APPROVED** [review]

**Comment by magicrub** [comment]
Looks good, thanks @Hwurzburg !

**Pull request merged**